### PR TITLE
Update to use current API in `wgpu-rs` master

### LINKED ISSUE #4: update to latest api
There's been quite a few changes since 0.2 crates and master wgpu-rs; are there any plans to upgrade this to the api that's on master any time soonish?

### PR DESCRIPTION
**Pull request by hecrj**
Fixes #4.

Most of the changes are quite straightforward. Let me know if something looks off!

### CODE REVIEW

File: src/pipeline.rs [hunk 1]
```diff
@@ -292,7 +289,7 @@ impl Pipeline {
             });
 
         render_pass.set_pipeline(&self.pipeline);
-        render_pass.set_bind_group(0, &self.uniforms);
+        render_pass.set_bind_group(0, &self.uniforms, &[]);
```
Comment (hecrj) [inline src/pipeline.rs]: I am not sure what the offsets here are supposed to be.

Comment (m4b) [reply src/pipeline.rs]: I usually pass &[] unless you're doing something unusual

**Comment by m4b** [comment]
Going to try this out right now :)

**Comment by m4b** [comment]
K i tested the example and it works with the git hash i'm using `ee40f4c574a4ca140b9d57ef66bc806f871abdad`; but I can't get text to render in my actual project, but I've never integrated it before so I'm probably just messing something up?

If i render into my swapchain view, I don't need to take into account model view transforms that apply to my vertices right? it should just write directly to the screen coordinates i give, yes?

**Comment by m4b** [comment]
Nevermind, I was drawing the glyph and then clearing the swapchain texture :man_facepalming:  :D this works great now!!!

**Comment by hecrj** [comment]
@m4b Awesome! We can merge this then :tada:

**Comment by m4b** [comment]
Thanks for the amazing velocity on updating this!!! :heart:

**Pull request merged**